doc/introduction.rst

```diff
@@ -0,0 +1,43 @@
+Introduction
+============
+
+XDG Desktop Portal is a session service that provides D-Bus interfaces for
+sandboxed apps. These D-Bus interfaces can be used by sandboxed apps without
+granting special permissions on their sandbox manifest.
```

Comment:
I know what you're referring to here but I think this will be confusing to people unaware that e.g flatpak has a manifest that can grant coarse access to host resources.

Maybe it's better to focus on the fact that with portals, permissions are granted or denied at runtime by the user?